authors/apps/authentication/migrations/0001_initial.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.1 on 2019-05-27 11:45
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('auth', '0009_alter_user_last_name_max_length'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='User',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('password', models.CharField(max_length=128, verbose_name='password')),
+                ('last_login', models.DateTimeField(blank=True, null=True, verbose_name='last login')),
+                ('is_superuser', models.BooleanField(default=False, help_text='Designates that this user has all permissions without explicitly assigning them.', verbose_name='superuser status')),
```

Comment:
Hey @neelxie compared to pep8 standards this looks so long, actually most of these lines here are abit long

Reply:
@jennizalwango Kindly note that this file and it's contents are auto generated when I run the migrations command; so I have no control over its standards.